package/recalboxOS/recalboxOS.mk

```diff
@@ -3,11 +3,16 @@
 # recalboxos
 #
 ################################################################################
-RECALBOXOS_RELEASE = 4.0.0-beta5
+RECALBOXOS_RELEASE = 4.0.0-beta6
 RECALBOXOS_SOURCE = $(RECALBOXOS_NAME)-$(RECALBOXOS_RELEASE).zip
 RECALBOXOS_SITE = https://github.com/recalbox/recalbox-os/releases/download/$(RECALBOXOS_RELEASE)
 
-ifeq ($(BR2_PACKAGE_RECALBOXOS_UNSTABLE),y)
+ifeq ($(BR2_PACKAGE_RECALBOXOS_STABLE),y)
+	RECALBOXOS_VERSION = 4.0.0
+	RECALBOXOS_EXTRA_DOWNLOADS = \
+		http://archive.recalbox.com/4/$(BR2_ARCH)/stable/last/boot.tar.xz \
```

Comment:
Okay, but i'm not sure how you reference to the 4.0.0 stable version before because it was released only a few days ago :D